test/health-food-test.js

```diff
@@ -60,50 +59,63 @@ describe('healthFood', function(){
     });
   });
 
-  context('moveHealthFood', function(){
-    var context = stub().of('beginPath').of('arc').of('closePath').of('fill');
+  // context('moveHealthFood', function(){
```

Comment:
We can keep these tests, commented out, so we will have them to reference in our next round.